modules/core/04-channel/keeper/grpc_query.go

```diff
@@ -96,26 +96,27 @@ func (q Keeper) ConnectionChannels(c context.Context, req *types.QueryConnection
 	channels := []*types.IdentifiedChannel{}
 	store := prefix.NewStore(ctx.KVStore(q.storeKey), []byte(host.KeyChannelEndPrefix))
 
-	pageRes, err := query.Paginate(store, req.Pagination, func(key, value []byte) error {
+	pageRes, err := query.FilteredPaginate(store, req.Pagination, func(key, value []byte, accumulate bool) (bool, error) {
```

Comment:
Same comment: maybe it's possible to modify one of the existing [tests](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel/keeper/grpc_query_test.go#L203)?